scripts/cnv_cromwell_tests/somatic/cnv_somatic_pair_wes_do-gc_workflow.json

```diff
@@ -4,7 +4,6 @@
   "CNVSomaticPairWorkflow.intervals": "/home/travis/build/broadinstitute/gatk/src/test/resources/large/cnv_somatic_workflows_test_files/ice_targets_sample-chr20.interval_list",
   "CNVSomaticPairWorkflow.normal_bam": "/home/travis/build/broadinstitute/gatk/src/test/resources/large/cnv_somatic_workflows_test_files/SM-74NEG-v1-chr20-downsampled.deduplicated.bam",
   "CNVSomaticPairWorkflow.normal_bam_idx": "/home/travis/build/broadinstitute/gatk/src/test/resources/large/cnv_somatic_workflows_test_files/SM-74NEG-v1-chr20-downsampled.deduplicated.bam.bai",
-  "CNVSomaticPairWorkflow.PreprocessIntervals.bin_length": 0,
```

Comment:
I would prefer to leave this in as illustration.  You have to have this parameter in for all exome runs.

Reply:
I think this actually fails because you did not update the test PoNs (which were built with bin_length != 0) accordingly.

I'm working on a branch that updates PreprocessIntervals and will update the test files and jsons accordingly there.